SU2_CFD/src/iteration/CFluidIteration.cpp

```diff
@@ -342,8 +336,15 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
     NewGridVel[iDim] = 0.0;
   }
 
-  GustDer = new su2double[3];
-  for (unsigned short i = 0; i < 3; i++) {
+  // Print some information to check that we are doing the right thing. Not sure how to convert the index back to a string...
+  if (rank == MASTER_NODE) cout << endl << "Setting up a wind gust type " << Gust_Type << " with amplitude of " << gust_amp << " in direction " << GustDir << endl;
+  if (rank == MASTER_NODE) cout << " U_inf      = " << Uinf << endl;
+  if (rank == MASTER_NODE) cout << " Physical_t = " << Physical_t << endl;
+  su2double loc_x = (xbegin + L + Uinf * (Physical_t - tbegin));
+  if (rank == MASTER_NODE) cout << " Location_x = " << loc_x << endl;
+
+  GustDer = new su2double[nDim+1];
+  for (unsigned short i = 0; i < nDim+1; i++) {
     GustDer[i] = 0.0;
   }
```

Comment:
```suggestion
  GustDer = new su2double[nDim+1]();
```

Reply:
Removed the loop for Gust and NewGridVel, too.

Reply:
Applied in 605839a